### PR TITLE
Made datetime attribute insert and match time-zone invariant

### PR DESCRIPTION
**Pull request by shiladitya-mukherjee**
## What is the goal of this PR?

Enables the BDD tests to check timezone-invariance of inserting and reading datetime attributes.

## What are the changes implemented in this PR?

- Added "set time-zone is: {time_zone_label}" which sets environment timezone to time_zone_label.
- Fixed datetime attribute parsing by bounding the number of splits of the attribute by ":" to 2.

### CODE REVIEW

File: test/behaviour/util/UtilSteps.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+package com.vaticle.typedb.core.test.behaviour.util;
+
+import io.cucumber.java.en.When;
+
+public class UtilSteps {
+    @When("set time-zone is: {word}")
+    public void set_timezone(String value){
+        System.setProperty("user.timezone", value);
+    }
+}
```
Comment (krishnangovindraj) [inline test/behaviour/util/UtilSteps.java:28]: Missing new line at the end of the file

File: .bazelrc [hunk 2]
```diff
@@ -20,4 +20,4 @@ try-import ./.bazel-remote-cache.rc
 build --incompatible_strict_action_env --java_language_version=11 --javacopt='--release 11' --@io_bazel_rules_docker//transitions:enable=false
 run --incompatible_strict_action_env
 test --incompatible_strict_action_env --test_env=PATH
-
+--java_runtime_version=remotejdk_11
```
Comment (krishnangovindraj) [inline .bazelrc:23]: I don't think this line was part of the ARM build fix. Please verify and remove.

**Review by krishnangovindraj — CHANGES_REQUESTED** [review]
Couple of small comments

We still need to wait for the M2 build fix to be merged before we merge this , OR rebase it to strip out those changes.

File: .bazelrc (hunk 2 above)
Comment (flyingsilverfin) [reply .bazelrc:23]: I think we actually want this :) let's keep it

Comment (krishnangovindraj) [reply .bazelrc:23]: Is this line alone valid syntax, or does it have to be:
`common --java_runtime_version=remotejdk_11`

File: .bazelrc [hunk 3]
```diff
@@ -17,7 +17,6 @@
 
 try-import ./.bazel-remote-cache.rc
 
-build --incompatible_strict_action_env --java_language_version=11 --java_runtime_version=remotejdk_11 --javacopt='--release 11' --@io_bazel_rules_docker//transitions:enable=false
-run --incompatible_strict_action_env
-test --incompatible_strict_action_env --test_env=PATH
-
+build --incompatible_strict_action_env --java_language_version=11 --javacopt='--release 11' --@io_bazel_rules_docker//transitions:enable=false --java_runtime_version=remotejdk_11
+run --incompatible_strict_action_env --java_runtime_version=remotejdk_11
+test --incompatible_strict_action_env --test_env=PATH --java_runtime_version=remotejdk_11
```
Comment (flyingsilverfin) [inline .bazelrc:22]: is there a 'common' block?

Comment (flyingsilverfin) [reply .bazelrc:22]: try putting it first. Move `--incompatible_strict_action_env` there as well if it works, then delete `run` since it'l be empty

Comment (shiladitya-mukherjee) [reply .bazelrc:22]: `Inherited 'common' options: --java_runtime_version=remotejdk_11 --incompatible_strict_action_env
ERROR: --java_runtime_version=remotejdk_11 :: Unrecognized option: --java_runtime_version=remotejdk_11`

**Review by flyingsilverfin — APPROVED** [review]

**Comment by flyingsilverfin** [comment]
Looks good! Please rebase onto the latest development and we can merge :)

**Comment by shiladitya-mukherjee** [comment]
> Looks good! Please rebase onto the latest development and we can merge :)

done!

**Pull request merged**